backend/inspirehep/search/mappings/v7/records/authors.json

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
could you remove the v5 mappings? hopefully git will detect that the files have been moved and show a proper diff (unless there are large changes)

Reply:
To review the mappings, I checked out your commit, then did
```
git checkout master -- backend/inspirehep/search/mappings/v5
git diff -w --no-index inspirehep/search/mappings/v5/records/hep.json inspirehep/search/mappings/v7/records/hep.json
```
which gives a fairly useable diff.